modules/atlas/splitmerge/main.nf

```diff
@@ -0,0 +1,36 @@
+process ATLAS_SPLITMERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::atlas=0.9.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas:0.9.9--h082e891_0':
+        'quay.io/biocontainers/atlas:0.9.9--h082e891_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(read_group_settings), path(blacklist)
+
+    output:
+    tuple val(meta), path("*.bam"), path("*.txt.gz"), emit: data
```

Comment:
1. Just saw you get a BAM output here too. It might be good to add the following check alongside the `defs` the `script` block:

https://nf-co.re/docs/contributing/modules#naming-conventions Point 5

2. Also is the txt.gz file defintely outputted as `.gz`? Just checking not missing a `gzip` command in the script below the command

Reply:
1. Sure, I'll do that
2. yes, it is: see here: https://bitbucket.org/wegmannlab/atlas/wiki/splitMerge